src/main/java/mil/dds/anet/resources/AttachmentResource.java

```diff
@@ -0,0 +1,134 @@
+package mil.dds.anet.resources;
+
+import com.codahale.metrics.annotation.Timed;
+import io.leangen.graphql.annotations.GraphQLArgument;
+import io.leangen.graphql.annotations.GraphQLMutation;
+import io.leangen.graphql.annotations.GraphQLQuery;
+import io.leangen.graphql.annotations.GraphQLRootContext;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.ResponseBuilder;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.core.StreamingOutput;
+import mil.dds.anet.AnetObjectEngine;
+import mil.dds.anet.beans.Attachment;
+import mil.dds.anet.beans.Person;
+import mil.dds.anet.database.AttachmentDao;
+import mil.dds.anet.utils.AnetAuditLogger;
+import mil.dds.anet.utils.AuthUtils;
+import mil.dds.anet.utils.DaoUtils;
+
+@Path("/api/attachment")
+public class AttachmentResource {
+
+  private final AttachmentDao dao;
+
+  public AttachmentResource(AnetObjectEngine engine) {
+    this.dao = engine.getAttachmentDao();
+  }
+
+  @GraphQLQuery(name = "attachment")
+  public Attachment getByUuid(@GraphQLArgument(name = "uuid") String uuid) {
+    return getAttachment(uuid);
+  }
+
+  @GraphQLQuery(name = "getAttachmentsForRelatedObject")
```

Comment:
Convention is not to name GraphQL accessors `getSomething` but just `something`, so `attachmentsForRelatedObject` in this case.
The method it calls is currently named `getAttachmentsOfRelatedObject` which should probably be `getAttachmentsForRelatedObject` to be aligned.

Second, I would also expect the related object's type to be passed here as a parameter.